source/_integrations/alpha_vantage.markdown

```diff
@@ -91,7 +91,7 @@ sensor:
         currency: USD
         symbol: GOOGL
     foreign_exchange:
-      - from: BTC
+      - name: BTC
```

Comment:
`name:` can be arbitrary and [set](https://www.home-assistant.io/integrations/alpha_vantage/#name) by the user. 

Ordering is irrelevant.